lib/stat_tracker.rb

```diff
@@ -0,0 +1,38 @@
+require 'CSV'
+# require './lib/game.rb'
+# require './lib/team.rb'
+# require './lib/game_teams.rb'
+
+class StatTracker
```

Comment:
This looks really great guys. Nice tight methods, looks to cover all the req's for Iteration 1.